ui/apps/platform/src/Containers/Administration/Events/AdministrationEventHint.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactElement } from 'react';
+import { List, ListItem, Text } from '@patternfly/react-core';
+
+const listItemRegExp = /^- /;
+
+type LineOrList = string | string[];
+
+function splitHint(hint: string): LineOrList[] {
+    const lineOrListArray: LineOrList[] = [];
+    const listItemArray: string[] = [];
+
+    function pushListItems() {
+        if (listItemArray.length !== 0) {
+            lineOrListArray.push([...listItemArray]);
+            listItemArray.length = 0;
+        }
+    }
+
+    hint.split('\n').forEach((line) => {
+        if (listItemRegExp.test(line)) {
+            listItemArray.push(line.slice(2));
+        } else {
+            pushListItems();
+            lineOrListArray.push(line);
+        }
+    });
+    pushListItems();
+
+    return lineOrListArray;
+}
+
+export type AdministrationEventHintProps = {
+    hint: string;
+};
+
+function AdministrationEventHint({ hint }: AdministrationEventHintProps): ReactElement {
+    /* eslint-disable no-nested-ternary */
+    /* eslint-disable react/no-array-index-key */
+    return (
+        <div>
+            {splitHint(hint).map((lineOrList, lineOrListIndex) =>
+                Array.isArray(lineOrList) ? (
+                    <List key={lineOrListIndex}>
+                        {lineOrList.map((listItem, listItemIndex) => (
+                            <ListItem key={listItemIndex} className="pf-u-mt-0">
+                                {listItem}
+                            </ListItem>
+                        ))}
+                    </List>
+                ) : lineOrList.length === 0 ? (
```

Comment:
Nit: at first glance I thought this was testing for an empty array, but the previous check in the ternary makes that impossible. What do you think about a `lineOrList === ''` check instead to make the string case more apparent?